Framework/CurveFitting/src/Functions/BackToBackExponential.cpp

```diff
@@ -41,17 +41,15 @@ void BackToBackExponential::init() {
 }
 
 /**
- * Get approximate height of the peak: function value at X0.
+ * Get approximate height (maximum value) of the peak (not at X0)
  */
 double BackToBackExponential::height() const {
-  double x0 = getParameter(3);
-  std::vector<double> vec(1, x0);
-  FunctionDomain1DVector domain(vec);
-  FunctionValues values(domain);
-
-  function(domain, values);
-
-  return values[0];
+  // height = I * ln2 /(FWHM - ln2 * S)
```

Comment:
Where does this analytic expression for the height come from?
Is it worth including in the BackToBackExponential.rst documentation file along with a comment that x0 may be the peak "position" but doesn't necessarily equal the maximum

Reply:
That's an expression I found to hold empirically on datasets that I simulated - I didn't attempt to derive it by differentiating the equation for the peak shape, maybe someone with mathmatica could prove/check it (but it seems to work OK)? I will add a comment to BackToBackExponential.rst as you suggest to note that in general X0 does not coincide with the maximum.

Reply:
I've checked again and the eq looks OK to me (p=ln2 in the attached graph for ~12000 simulated datasets with reasonable parameters)
![height_B2Bexp](https://user-images.githubusercontent.com/55979119/98541064-c1f5b700-2286-11eb-8464-17c3744ed17a.png)

Reply:
OK I see. So that plot is for I fixed at 1? And you've simulated various values of A, B and S?

If the formula for the height is empirical, is the same true for the fwhm formula where you add two terms together (2nd term is fwhm of a gaussian and am guessing the first term is the width of the two exponentials). Or if it's not empirical, it's by "intuition" where you've added the widths of the exponentials and the gaussian components.

Reply:
Yep that's correct, the above graph was for simulated data with I=1. The second term in the equation for FWHM(S,A,B) is indeed the Gaussian, I found the first term by looking at what function fitted the difference between the actual fwhm and the Gaussian (normalised by the fwhm of the back to back exponential that seems to be the magic number for getting the fwhm to collapse onto the same curve) - with the constraint that it had to give the right width as S tended to 0 and S tends to large (>200). I was expecting the widths to add in quadrature (as per usual for convoluted gaussians), but the back to back exponential is not that helpful. I can show similar graphs for fwhm vs S.